data/binding/binditems.go

```diff
@@ -39,35 +45,50 @@ func (b *boundBool) Get() bool {
 }
 
 func (b *boundBool) Set(val bool) {
-	if *b.val == val {
-		return
-	}
 	if b.val == nil { // was not initialized with a blank value, recover
 		b.val = &val
 	} else {
+		if *b.val == val {
+			return
+		}
+
 		*b.val = val
 	}
 
 	b.trigger()
 }
 
+func (b *boundBool) Reload() {
+	if b.val == nil {
```

Comment:
Can this ever be nil?

Reply:
Yes. As soon as you allow developers to pass in the pointer it can be nil

Reply:
I don't think this is true as the pointer was checked by the constructor

Reply:
Yeah. There was another way to construct at some point, and I missed that it could no longer happen